### DISCUSSION TITLE
Are there any hopes of a Flatpak version?

### DISCUSSION THREAD

**Discussion by cyrus01337** [pear-devs/pear-desktop#2267] [General]
Purely curious and was something I thought about as I was writing a small auto-update script that works on Fedora, are there any considerations for this/has this been raised before elsewhere that I might have missed?

**Comment by ArjixWasTaken** [comment]
Looks like we share the trait of being blind :rofl:

https://github.com/th-ch/youtube-music/pull/2047

**Reply by cyrus01337** [reply]
Oh brother, might need a new prescription after all. Thanks for this!
